src/timer-view/timer-view.html

```diff
@@ -14,7 +14,7 @@
     [[prompt]]
     <div>
       [[secondsRemaining]]
-      <iron-timer id="timer" start-time="1" current-time="{{secondsRemaining}}" on-iron-timer-end="_timerEnd"></iron-timer>
```

Comment:
iron-timer has a startTime and endTime, but endTime won't work if the user navigates to a new tab, so that is why we used a secondsRemaining. I will look to see if there is a better way to do this.